tests/core/test_validation.py

```diff
@@ -85,3 +85,48 @@ def test_validate_install_cli_args_rejects_invalid_relative_paths(
 
     with pytest.raises(InstallError):
         validate_install_cli_args(args)
+
+
+@pytest.mark.parametrize(
+    "left,right",
+    (
+        (
+            "erc1319://0x6b5DA3cA4286Baa7fBaf64EEEE1834C7d430B729:1",
+            "erc1319://0x6b5DA3cA4286Baa7fBaf64EEEE1834C7d430B729:1",
+        ),
+        (
+            "erc1319://0x6b5DA3cA4286Baa7fBaf64EEEE1834C7d430B729:1/dai?version=1.0.0",
+            "erc1319://0x6b5DA3cA4286Baa7fBaf64EEEE1834C7d430B729:1",
```

Comment:
do we support ENS names in these?

Reply:
in theory yes, in reality, not until I make an update to ethpm, but that pr is coming soon